flocker/node/agents/blockdevice.py

```diff
@@ -944,6 +944,42 @@ def get_device_path(blockdevice_id):
         """
 
 
+class IProfiledBlockDeviceAPI(IBlockDeviceAPI):
+    """
+    An interface for ``IBlockDeviceAPI`` implementers that are capable of
+    creating volumes with a specific profile.
+    """
+
```

Comment:
Version numbers as attributes of implementations would allow us to... change `IProfiledBlockDeviceAPI` within our code rather than copy/paste alter for new versions. But that means we don't have good way to refer to old version in any adapter code, so seems better to just copy/paste for this particular use case.